Tiltfile

```
@@ -0,0 +1,62 @@
+# Load config setup
+load_dynamic("./development/tiltfiles/setup.tiltfile")
+
+load('ext://restart_process', 'docker_build_with_restart')
+
+docker_build_with_restart(
+  "fury-ui-backend:local-dev",
+  ".",
+  dockerfile="Dockerfile",
+  target="development",
+  live_update=[
+    sync("./internal", "/app/internal"),
+    sync("./go.mod", "/app/go.mod"),
+    sync("./go.sum", "/app/go.sum"),
+  ],
+  build_args={
+    "PORT": os.getenv("PORT"),
+  },
+  entrypoint=["go", "run", "main.go"]
+)
+
+docker_build(
+  "fury-ui-frontend:local-dev",
+  "./web-client",
+  dockerfile="./web-client/Dockerfile",
+  target="nginx-dev",
+  live_update=[
+    sync("./web-client", "/app/web-client"),
+  ],
+  build_args={
+#    "PORT": os.getenv("PORT"),
+  }
+)
+
+k8s_yaml(
+  helm(
+    './helm_chart',
+    name='fury-ui',
+    namespace='fury-ui',
+    values='development/helm/values.yaml',
+#    set=['config.controlPlaneAddress=' + os.getenv("CONTROL_PLANE_ADDRESS")]
+    set=[]
+  )
+)
+
+k8s_resource(
+  workload="fury-ui-backend",
+  links=[
+    link("https://fury-ui.dev/", "fury-ui-backend"),
```

perhaps it should point at the api endpoint, or to a swagger ui?
```suggestion
    link("https://fury-ui.dev/api/v1", "fury-ui-backend"),
```